src/libs/FileSorter.h

```diff
@@ -47,11 +45,11 @@ class FileSorter {
 
     string directory_path;
     bool error;
+    bool sorted;
```

Comment:
with bools we usually do this...
```
struct {
  bool error:1;
  bool sorted:1;
};
```
and move it to the end of the class. This saves memory.